src/tool/subcommands/api_cmd.rs

```diff
@@ -980,7 +980,12 @@ fn state_tests_with_tipset<DB: Blockstore>(
             tests.extend([
                 RpcTest::identity(StateReplay::request((tipset.key().into(), msg_cid))?),
                 validate_message_lookup(
-                    StateWaitMsg::request((msg_cid, 0))?.with_timeout(Duration::from_secs(30)),
```

Comment:
Can we test v0 as well, or is it not feasible?

Reply:
Not easily feasible, the actual V0 method name uses a `V0` suffix and does not exist in lotus

Reply:
that's disappointing. It effectively makes the entire v0 path for this method untested and likely incorrect (now or in the future)

Reply:
Right now, we can either test the v0 methods or the v1 methods but not both. Aatif is working on fixing this.